bpmn-js-app/angular.json

```diff
@@ -51,6 +52,10 @@
                   "type": "initial",
                   "maximumWarning": "2mb",
                   "maximumError": "5mb"
+                },
```

Comment:
What does this do? How does this contribute / improve to the example?

Reply:
@nikku: it is generated automatically by `ng update @angular/core`